doc/userguide/install.rst

```diff
@@ -149,21 +149,27 @@ For Debian 10 (buster), for instance, run the following as ``root``::
 Fedora
 ^^^^^^
 
+The following is an example of install Suricata 6.0 on Fedora. If you wish to install 5.0 or 4.1 instead, change the version in *@oisf/suricata-6.0*.
```

Comment:
s/install/installing

Reply:
corrected and committed to the branch, not sure if I need to open a new PR or not though.